downloads/swan-lake-release-notes/swan-lake-2201.3.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,468 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.3.0 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-3-0/
+active: 2201-3-0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-3-0
+    - /downloads/swan-lake-release-notes/2201.3.0/
+    - /downloads/swan-lake-release-notes/2201-3-0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-3-0-swan-lake
+---
+
+## Overview of Ballerina 2201.3.0 (Swan Lake)
+
+<em>2201.3.0 (Swan Lake) is the third major release of 2022, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R4 version of the Language Specification.</em> 
+
+## Update Ballerina
+
+**If you are already using Ballerina 2201.0.0 (Swan Lake)**, run either of the commands below to directly update to 2201.3.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/).
+
+`bal dist update` (or `bal dist pull 2201.3.0`)
+
+**If you are using a version below 2201.0.0 (Swan Lake)**, run the commands below to update to 2201.3.0.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+
+2. Run `bal dist update` ( or `bal dist pull 2201.3.0`) to update your Ballerina version to 2201.3.0.
+
+However, if you are using a version below 2201.0.0 (Swan Lake) and if you already ran `bal dist update` (or `bal dist pull 2201.3.0`) before `bal update`, see [Troubleshooting](/downloads/swan-lake-release-notes/swan-lake-2201.0.0#troubleshooting) to recover your installation.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
+### New features
+
+#### Added support for function types with defaultable parameters
+
+Default values are now allowed for parameters of function types.
+
+```ballerina
+import ballerina/io;
+
+public function main() {
+    int num1 = 100;
+    int num2 = 50;
+    int num3 = 25;
+    function (int a = 0, int b = 0, int c = 0) returns int total = getSum;
+
+    io:println(total()); // Prints `0`.
+    io:println(total(num1, num2)); // Prints `150`.
+    io:println(total(num1, num2, num3)); // Prints `175`.
+}
+
+function getSum(int num1, int num2, int num3) returns int {
+    return num1 + num2 + num3;
+}
+```
+
+#### Added a new field to the `display` annotation
+
+A new field named `kind` has been introduced to the `display` annotation to indicate the kind of data. Allowed values are `text`, `password`, and `file`.
+
+```ballerina
+public type RefreshTokenGrantConfig record {|
+    @display {
+        iconPath: "Field.icon",
+        label: "clientSecret field",
+        kind: "password"
+    }
+    string clientSecret;
+|};
+```
+
+### Improvements
+
+#### More improvements on working with optional fields
```

Comment:
Since this is under `Improvements`  title do we need to say `More improvements...`. Instead should we be more precise on what type of improvements on optional fields? @MaryamZi / @hasithaa

Reply:
I think it's a bit difficult to include all the changes as a concise title. We'll check again anyway.

Reply:
@MaryamZi - Can we finalize this, please?

Reply:
Updated via https://github.com/ballerina-platform/ballerina-dev-website/pull/5937/commits/94e875a8a11fbca9a1ff74d27c690b33495c6de4.